### PR TITLE
[MNOE-777] Extra max auto-height for widgets & centered arrow

### PR DESCRIPTION
**Pull request by enizor**
10 users (larger cells) did not fit in the previous value.

### CODE REVIEW

**Comment by enizor** [comment]
A secon commit will arrive with a centered sortable-table arrow

**Comment by enizor** [comment]
Using centered icons: 
- Before
image image
- After
 imageimage

File: src/components/sortable-table/sortable-table.less [hunk 1]
```diff
@@ -19,12 +19,12 @@ mno-sortable-table {
 
     .st-sort-ascent:after{
       // Content code for 'fa-sort-asc'
```
Comment (alexnoox) [inline src/components/sortable-table/sortable-table.less]: Update comment `// Content code for 'fa-caret-up'`

File: src/components/sortable-table/sortable-table.less [hunk 2]
```diff
@@ -19,12 +19,12 @@ mno-sortable-table {
...
-      content: "\f0de";
+      content: "\f0d8";
     }
 
     .st-sort-descent:after{
       // Content code for 'fa-sort-desc'
```
Comment (alexnoox) [inline src/components/sortable-table/sortable-table.less]: Update comment `// Content code for 'fa-caret-down'`

**Review by alexnoox — CHANGES_REQUESTED** [review]

**Comment by alexnoox** [comment]
Please also commit a build as the last commit 😄

**Review by alexnoox — APPROVED** [review]

**Comment by alexnoox** [comment]
👍

**Pull request merged**